scripts/deploy.py

```diff
@@ -92,6 +92,7 @@ def main():
 
     rewards = get_address("Rewards contract", default="treasury.ychad.eth")
     guardian = get_address("Vault Guardian", default="dev.ychad.eth")
+    management = get_address("Vault Management", default="ychad.eth")
```

Comment:
@fubuloubu @dougstorm we need an ens for sms

Reply:
`brain.ychad.eth`!